src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java

```diff
@@ -1521,17 +1521,47 @@ public void enterIntoFieldWithLabel(String labelText, String text) {
      * Goes to the next tab.
      */
     public void goNextTab() {
-        String oldTab = webDriver.getWindowHandle();
-        List<String> newTab = new ArrayList<String>(webDriver.getWindowHandles());
-        newTab.remove(oldTab);
-        int numTabs = newTab.size();
-        if (numTabs > 2) {
-            String message = "Too many tabs. Expected 2 but there are " + numTabs;
+        List<String> allTabs = new ArrayList<String>(webDriver.getWindowHandles());
+
+        // we've removed 1 tab already - there should only be one other tab.
+        if (allTabs.size() > 2) {
```

Comment:
Should this not be equal to or greater than 2

Reply:
no because we want it to be 2 tabs which we switch to on line 1562